Source/WebCore/platform/network/BlobPart.h

```diff
@@ -27,6 +27,10 @@
 
 #include <wtf/URL.h>
 
+namespace IPC {
+template<typename T, typename> struct ArgumentCoder;
```

Comment:
Out of curiosity, where did this one come from? (Also, you probably don't need the explicit `T`.)